src/generators.rs

```diff
@@ -1933,6 +1933,16 @@ pub fn gen_contract_call(scale: u16) -> (Option<String>, String) {
     (None, body)
 }
 
+fn gen_contract_of(scale: u16) -> (Option<String>, String) {
+    let mut body = String::new();
+
+    for _ in 0..scale {
+        body.push_str("(contract-call? .use-trait bench-contract-of .impl-trait) ");
```

Comment:
Could you explain what's happening in this line of clarity code?

Reply:
I'm contract-calling a function that gets deployed during `setup_marf` called `bench-contract-of` with the arg `.impl-trait`, which is the principal of a contract that implements `trait-1`.

`bench-contract-of` is in a contract called `.use-trait` that looks like:

```
(use-trait trait-1 .define-trait.trait-1)

(define-public (bench-contract-of (contract <trait-1>))
    (ok (contract-of contract)))
```

All this function does is invoke `contract-of`, which I believe can only be used from within a function since it requires a trait reference.

Reply:
Ohh I see. Could you rename `.impl-trait` and `.use-trait` - I think it's confusing bc they are also keywords in clarity. 

Maybe like `.impl-trait-principal` and `.use-trait-contract`.